docs/extensions/MouseCursor.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Mouse.Cursor attached property
+author: martinsuchan
+ms.date: 11/05/2017
+description: Mouse.Cursor attached property enables you to easily change the mouse cursor over specific Framework elements.
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, Mouse, cursor, extensions
+---
+
+# Mouse.Cursor attached property
+
+The **Mouse.Cursor attached property** enables you to easily change the mouse cursor over specific Framework elements.
+
+## Example
+
+First we need to add the namespace declaration, in this case we added to the root Page element:
+```xaml
+xmlns:ui="using:Microsoft.Toolkit.Uwp.UI.Extensions"
+```
+
+Then we can use the attached property on any FrameworkElement in XAML:
+```xaml
+ui:Mouse.Cursor="Hand"
+```
+
+This is how it looks in full Page context:
+```xaml
+<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.MouseCursorPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"    
+    xmlns:ui="using:Microsoft.Toolkit.Uwp.UI.Extensions">
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+        <Border ui:Mouse.Cursor="Hand"
+	        Width="220" Height="120" Background="DeepSkyBlue"
+	        HorizontalAlignment="Center" VerticalAlignment="Center"/>
+    </Grid>
+</Page>
+```
+
+## Note
+Even though Microsoft recommends in [UWP Design guidelines](https://docs.microsoft.com/en-us/windows/uwp/input-and-devices/mouse-interactions#cursors) hover effects instead of custom cursors over interactive elements, custom cursors can be useful in some specific scenarios.
+
+## Limitations
+Because the UWP framework does not support metadata on Attached Properties, specifically the [FrameworkPropertyMetadata.Inherits](https://msdn.microsoft.com/en-us/library/ms557301%28v=vs.110%29.aspx) flag, the Mouse.Cursor might not work properly in some very specific XAML layout scenarios when combining nested FrameworkElements with different Mouse.Cursor values set on them.
```

Comment:
Example?

Reply:
Imagine three Border elements A inside B inside C. A is smaller than B which is smaller than C. A has Mouse.Cursor="Hand", B has the default cursor and C has Mouse.Cursor="Cross" set.
The expected behavior is Cross Cursor shown over Border B and C, and Hand over A. It would work like that in WPF with **FrameworkPropertyMetadata.Inherits** flag, but in the UWP app it changes the Hand cursor to Arrow when moving from A to B, because the inheritance is not working as expected.
If you think I should include this broken sample in the lib then I'll add it as well.

Reply:
In that case, could we not add code to look for the property on the parent walking up the tree to have it behave as you'd expect?

Reply:
Right now the cursor is changed to the Mouse.Cursor value of the element under the cursor when the PointerExited event is triggered. It's the best solution available right now.

Reply:
Thanks cool thanks